Signal/src/ViewControllers/MediaPageViewController.swift

```diff
@@ -801,7 +803,7 @@ class CaptionView: UIView {
 
     class CaptionTextView: UITextView {
 
-        var kMaxHeight: CGFloat = ScaleFromIPhone5(100)
+        var kMaxHeight: CGFloat = ScaleFromIPhone5(200)
```

Comment:
I'd accidentally lowered this to 100 while testing the scroll view, but I think this is closer to where we'd want to be.